packages/analytics/analytics-metric-provider/src/components/display/MetricsCard.vue

```diff
@@ -0,0 +1,392 @@
+<template>
+  <div class="metricscard">
+    <div
+      class="metricscard-title"
+      :class="cardSize"
+    >
+      <component
+        :is="iconMap.get(cardType)"
+        v-if="!hideTitleIcon"
+        class="metricscard-icon"
+        :color="KUI_COLOR_TEXT_NEUTRAL"
+        :size="KUI_ICON_SIZE_30"
+      />
+      <component :is="titleTag">
+        {{ title }}
+      </component>
+      <KTooltip
+        v-if="tooltip"
+        class="metricscard-tooltip"
+        placement="right"
+        :text="tooltip"
+      >
+        <InfoIcon
+          :color="KUI_COLOR_TEXT_NEUTRAL"
+          :size="KUI_ICON_SIZE_30"
+        />
+      </KTooltip>
+    </div>
+    <div
+      v-if="description && cardDisplayFull"
+      class="metricscard-description"
+    >
+      <span>{{ description }}</span>
+    </div>
+    <!-- TODO: remove outer div once size variant no longer needed - MA-2193  -->
+    <div
+      class="metricscard-valuetrend"
+    >
+      <!-- Metric value - error -->
+      <div
+        v-if="hasError"
+        class="metricscard-error"
+      >
+        <WarningIcon
+          :color="KUI_COLOR_TEXT_NEUTRAL"
+          :size="KUI_ICON_SIZE_40"
+        />
+        <div>&nbsp;{{ errorMessage }}</div>
+      </div>
+      <!-- Metric value -->
+      <div
+        v-else
+        class="metricscard-value"
+        :class="cardSize"
+        data-testid="metric-value"
+      >
+        {{ metricValue }}
+      </div>
+      <div
+        v-if="cardDisplayFull"
+        class="metricscard-trend"
+      >
+        <!-- Trend icon and percentage change -->
+        <div
+          class="metricscard-trend-change"
+          :class="textColor(changePolarity)"
+          data-testid="metric-trend-parent"
+        >
+          <!-- Trend Icon -->
+          <component
+            :is="trendIcon"
+            v-if="changePolarity !== 0"
+            :color="colorAttribute(changePolarity)"
+            :size="KUI_ICON_SIZE_30"
+          />
+          <!-- No change icon -->
+          <EqualIcon
+            v-else
+            :color="KUI_COLOR_TEXT_NEUTRAL_STRONG"
+            :size="KUI_ICON_SIZE_30"
+          />
+          <div data-testid="metric-trend-change">
+            {{ metricChange }}
+          </div>
+        </div>
+        <!-- Trend range text -->
+        <div
+          v-if="trendRange"
+          class="metricscard-trend-range"
+        >
+          {{ trendRange }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { PropType } from 'vue'
+import {
+  KUI_COLOR_BORDER_DANGER_STRONG,
+  KUI_ICON_SIZE_30,
+  KUI_ICON_SIZE_40,
+  KUI_COLOR_TEXT_SUCCESS, // Positive trend
+  KUI_COLOR_TEXT_NEUTRAL_STRONG, // Neutral trend
+  KUI_COLOR_TEXT_NEUTRAL,
+} from '@kong/design-tokens'
```

Comment:
Doesn't look like it's just a dev dependency

Reply:
> since the code is compiled at build time, those sass variables are changed into raw values, meaning they don’t need to be packaged

I think we're good here, keeping it as devDep